### DISCUSSION TITLE
FAIL - Application at context path [/WebViewerExample] could not be started

### DISCUSSION THREAD

**Discussion by ghost** [eclipse-birt/birt#1691] [Q&A] [answered]
When i try to deploy my birt runtime folder in Tomcat it gives an error as per the img attached
![img](https://github.com/eclipse-birt/birt/assets/167118024/abfda7e9-2777-4d72-8f60-a1e729ee09ba)

**Comment by speckyspooky** [comment]
Which versions do you use for BIT, Java and Tomcat?
What is noted in your Tomcat-log-file?

**Reply by ghost** [reply]
all latest

**Reply by merks** [reply]
No one can guess from a picture.  Without a log, not much can be done. Also all latest is rather unspecific. I doubt you are using birt 4.16 or Java 22.0.2.

**Reply by ghost** [reply]
am using birt 4.15 and java 22.

**Reply by speckyspooky** [reply, marked as answer]
But you must be aware that the Tomcat must be a 9.x.x version not a 10.x.x version.
Thefore it is interesting which Tomcat version is used. BIRT is runable under Tomcat 9.x.x and not with Tomcat 10.x.x

**Reply by ghost** [reply]
Thanks, but when i place but .rptdesign file in the report folder of WebViewerExample and access it through the url of "http://localhost:8080/WebViewerExample/frameset?__report=TAS.rptdesign" it displays  The report file : TAS.rptdesign does not exist or contains errors.

**Reply by ghost** [reply]
@speckyspooky @merks pls help

**Reply by merks** [reply]
It’s Friday evening 22:00 here. 

You really must share log details.

**Reply by ghost** [reply]
@merks i have solved that issue

**Reply by speckyspooky** [reply]
Great!

**Comment by ghost** [comment]
ok thanks let me try and let you know
